_posts/en/newsletters/2023-02-08-newsletter.md

```diff
@@ -0,0 +1,130 @@
+---
+title: 'Bitcoin Optech Newsletter #237'
+permalink: /en/newsletters/2023/02/08/
+name: 2023-02-08-newsletter
+slug: 2023-02-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a discussion about storing data in
+transaction witnesses and references a conversation about mitigating LN
+jamming.  Also included are our regular sections with the summary of a
+Bitcoin Core PR Review Club meeting, announcements of new releases and
+release candidates, and descriptions of notable changes to popular
+Bitcoin infrastructure software.
+
+## News
+
+- **Discussion about storing data in the block chain:** users of a new
+  project recently began storing large amounts of data in the witness
+  data for segwit v1 ([taproot][topic taproot]) transactions.   Robert
```

Comment:
why is the link wrapped in parentheses in this case?

Reply:
To me, that looks reasonable, since it's saying segwit v1, and taproot is just another name for segwit v1 (to help anyone who might not be aware of that fact).

Reply:
Ah, I thought it was for markup reasons.
i'm trying to figure out why my PR [#1001](https://github.com/bitcoinops/bitcoinops.github.io/pull/1001) isn't passing CI, and it has something to do with [text][topic x] not converting to html properly.